src/index.js

```diff
@@ -44,6 +44,27 @@ export default class Kinto {
   }
 
   /**
+   * Creates a remote transformer constructor, the ES5 way.
+   *
+   * @return {RemoteTransformer}
+   */
+  static createRemoteTransformer(proto) {
+    if (!proto || typeof proto !== "object")
+      throw new Error("Expected prototype object.");
```

Comment:
I though it was a good practice to always enclose iffed instructions inside a block `{` and `}`.